software/feather-wiki.yml

```diff
@@ -0,0 +1,17 @@
+name: "Feather Wiki"
+website_url: "https://feather.wiki"
+source_code_url: "https://codeberg.org/Alamantus/FeatherWiki"
+description: "A lightning fast and infinitely extensible tool for creating personal non-linear notebooks, databases, and wikis that is entirely self-contained, runs in your browser, and is only 58 kilobytes in size."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Javascript
+tags:
+  - Wikis
+  - Static Site Generators
```

Comment:
`ERROR:awesome_lint.py: Feather Wiki: tag Static Site Generators points to a tag which redirects to another list.`

The Static Site Generator category is explicitely delegated to to other sites, please remove this.